code/API_definitions/geofencing.yaml

```
@@ -63,7 +63,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.1.0-rc2
+  version: 0.2.0-wip
```

```suggestion
  version: 0.1.0-wip
```

We have to fix this for 0.1.0 as well so let's keep the current version. We'll have to create an rc3 probably